src/polyswarm_api/__init__.py

```diff
@@ -116,8 +58,12 @@ def __init__(self, key, uri="https://api.polyswarm.network/v1", get_limit=100,
 
         self.get_semaphore = asyncio.Semaphore(get_limit)
 
-        # ...sigh
-        self.engine_resolver = EngineResolver(self.network)
+        self.network = "prod" if uri.find("lb.kb") == -1 else "lb.kb"
```

Comment:
`self.network` here should be 'stage' I believe, to remain consistent with line 104?